bin/cli.js

```diff
@@ -0,0 +1,235 @@
+#!/usr/bin/env node
+'use strict';
+
+const program = require('commander');
+const common = require('metarhia-common');
+const karma = require('karma');
+const yaml = require('yaml').default;
+const path = require('path');
+const fs = require('fs');
+
+const splitOpts = v => v.split(',');
+
+const cliOptions = [
+  [ '--exclude <patterns>',      'Exclude tests patterns', splitOpts, [] ],
+  [ '--browsers <values>',       'Browsers to run', splitOpts, [] ],
+  [ '--reporter <value>',        'Reporter name'],
+  [ '--log-level <value>',       'Log level'],
+  [ '-p, --browser-port <n>',    'Browser port' ],
+  [ '-c, --config <value>',      'Config file' ],
+];
+
+const karmaLogLevels = {
+  'disable': [ 'LOG_DISABLE', 'none' ],
+  'error':   [ 'LOG_ERROR', 'errors-only' ],
+  'warn':    [ 'LOG_WARN', 'minimal' ],
+  'info':    [ 'LOG_INFO', 'normal' ],
+  'debug':   [ 'LOG_DEBUG', 'verbose' ],
+};
+
+const browserLaunchers = {
+  'Chrome':         'karma-chrome-launcher',
+  'ChromeHeadless': 'karma-chrome-launcher',
+  'Firefox':        'karma-firefox-launcher',
+  'IE':             'karma-ie-launcher',
+  'Opera':          'karma-opera-launcher',
+  'Safari':         'karma-safari-launcher',
+};
+
+const logLevels = ['quiet', 'disable', 'error', 'warn', 'info', 'debug'];
+
+const isLogAtLeast = (level1, level2) => {
+  if (logLevels.indexOf(level1) === -1) level1 = 'default';
+  if (logLevels.indexOf(level2) === -1) level2 = 'default';
+  return logLevels.indexOf(level1) >= logLevels.indexOf(level2);
+};
+
+const merge = (arr1 = [], arr2 = []) => common.merge(arr1, arr2);
+
+const exclude = (files, filterArr) =>
+  filterArr.map(path => path.replace('.', '\\.')
+    .replace('*', '.+')
+    .replace('?', '.'))
+    .map(path => new RegExp(path))
+    .reduce((files, regexp) => files.filter(file => !regexp.test(file)), files);
+
+const parseFile = file => {
+  const data = fs.readFileSync(path.resolve(file), 'utf8');
+  switch (common.fileExt(file)) {
+    case 'json':
+      return JSON.parse(data);
+    case 'yml':
+    case 'yaml':
+      return yaml.parse(data);
+    default:
+      return {};
+  }
+};
+
+const loadFiles = files => {
+  const result = [];
+  files
+    .map(file => {
+      if (fs.existsSync(file + '.js')) {
+        return file + '.js';
+      } else if (fs.existsSync(file)) {
+        return file;
+      } else {
+        console.error('File does not exist:', file);
+        process.exit(1);
+        return '';
+      }
+    })
+    .forEach(file => {
```

Comment:
Nit: I think we usually put chained calls on the same line, though that's not required.